identity/sharedid.md

```diff
@@ -18,7 +18,17 @@ As of Prebid.js 5.0, PubCommon ID is no longer supported -- it's been merged int
 
 SharedId is a convenient Prebid-owned first party identifier within the [Prebid UserId Module framework](/dev-docs/modules/userId.html).
 
-## How does it work?
+There are multiple ways to integrate SharedId on your site. See the table below for a breakout of options, and the rest of this document for detailed integration instructions.
+
+{: .table .table-bordered .table-striped }
+| Implementation | Description | Cookie Lifetime | SAFARI COOKIE LIFETIME | TECHNICAL DIFFICULTY | REVENUE BENEFIT |
+| --- | --- | --- | --- | --- | --- |
+| 3rd Party Cookie Only | No first party cookie solution. | Some Blocked | Blocked | None | Low |
+| Prebid User Id | Including User Id Module in your Prebid installation. | 365 days | 7 days | Basic | Good |
+| PubCID Script | Adding the PubCID script to work with our ad code or a custom integration. | 365 days | 7 days | Basic | Good |
```

Comment:
This is legacy method one might use without prebid, no longer recommended. 

I would change Description to 
'Adding the legacy PubCID script; not maintained by Prebid.org'
 Technical Difficulty to 'High' and Revenue to 'Varies'

Reply:
done